src/main/java/com/softserve/academy/spaced/repetition/domain/Account.java

```diff
@@ -19,9 +25,15 @@
     private Long id;
 
     @Column(name = "password", nullable = false)
+    @NotNull(message = NULL_MESSAGE, groups = Request.class)
+    @Size(message = PASSWORD_SIZE_MESSAGE, min = PASSWORD_MIN_SIZE, max = PASSWORD_MAX_SIZE, groups = Request.class)
     private String password;
 
     @Column(name = "email", unique = true, nullable = false)
+    @NotNull(message = NULL_MESSAGE, groups = Request.class)
+    @Size(min = 1, message = EMPTY_MESSAGE, groups = Request.class)
```

Comment:
I was wrong that NotEmpty can be replaced by Size(min=1), as user will be able to enter one space.

NotEmpty can be replaced by Pattern("\S+").

In this case there is no need for Size as the Pattern annotation will do the check.

Reply:
@antonbakalets with one space @Size will return that value is valid, but @Pattern - is invalid.

Reply:
@antonbakalets I must delete Size?

Reply:
No, it was an optional comment.